src/main/java/org/zalando/planb/provider/CassandraProperties.java

```diff
@@ -17,6 +19,8 @@
     private int port = ProtocolOptions.DEFAULT_PORT;
     private ConsistencyLevel writeConsistencyLevel = ONE; // TODO what is a sane default?
```

Comment:
I still consider ONE only a last resort, personally I would stat of with LOCAL_QUORUM